luzer/CMakeLists.txt

```diff
@@ -36,6 +36,12 @@ target_compile_options(luzer_impl PRIVATE
     -pedantic
     -fsanitize=fuzzer-no-link
 )
+if(LUA_HAS_JIT)
```

Comment:
I'm confused by this option. Should it be set separately from `-DENABLE_LUAJIT`?

Reply:
This is an internal variable. Yet another way to set Lua library is setting LUA_LIRBARY and LUA_INCLUDE_DIR outside (we use this way in lua-c-api-tests). `LUA_HAS_JIT` is used when LuaJIT library is set.